common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1463,21 +1488,41 @@ object CelebornConf extends Logging {
       .checkValue(Seq("roundrobin", "loadaware").contains(_), "")
       .createWithDefault("roundrobin")
 
-  def initialPartitionSize(conf: CelebornConf): Long = {
-    Utils.byteStringAsBytes(conf.get("rss.initial.partition.size", "64m"))
-  }
+  val INITIAL_ESTIMATED_PARTITION_SIZE: ConfigEntry[Long] =
+    buildConf("celeborn.worker.initialEstimatedPartitionSize")
```

Comment:
This config is not intended for the worker. May be the "celeborn.master" will be a better namespace?

Reply:
Forgot change. ==